internal/cluster/cluster_test.go

```diff
@@ -1171,7 +1171,7 @@ var _ = Describe("VerifyRegisterHost", func() {
 		id                 strfmt.UUID
 		clusterApi         *Manager
 		preInstalledError  string = "Host can register only in one of the following states: [insufficient ready pending-for-input adding-hosts]"
-		postInstalledError string = "Cannot add host to a cluster that is already installed, please use the day2 cluster option"
+		postInstalledError string = "Cannot add hosts to an existing cluster using the original Discovery ISO. Try to add new hosts by using the Discovery ISO that can be found in console.redhat.com under your cluster “Add hosts” tab."
```

Comment:
```suggestion
		postInstalledError string = "Cannot add hosts to an existing cluster using the original Discovery ISO. Try adding new hosts by using the Discovery ISO found in console.redhat.com under your cluster “Add hosts” tab."
```

Reply:
I am using the text approved by @nirfarkas in the Jira ticket